src/ringme.js

```diff
@@ -177,15 +177,15 @@ var RingMe = new function() {
    * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
    * SOFTWARE.
    */
-  var _launchUri = function(uri, successCallback, noHandlerCallback, unknownCallback) {
```

Comment:
There's a syntax error here, it is missing the attribution and the `function` keyword. This should be something like
```
const _lauchUri = function(uri, successCallback, noHandlerCallback, unknownCallback) {
```